test/cql-pytest/test_select_mutation_fragments.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests concerning the SELECT MUTATION FRAGMENT statement, which allows dumping
+# the underlying mutation fragment data stream, for a table.
+
+
+import pytest
+import util
+import cassandra.protocol
+
+
+@pytest.fixture(scope="module")
+def test_table(cql, test_keyspace):
+    """ Prepares a table for the mutation dump tests to work with."""
+    with util.new_test_table(cql, test_keyspace, 'pk1 int, pk2 int, ck1 int, ck2 int, v text, s text static, PRIMARY KEY ((pk1, pk2), ck1, ck2)') as table:
+        yield table
+
+
+def test_mutation_dump_order_by(cql, test_table):
```

Comment:
I know this is just an early version, but eventually please add tests also for the success cases. For example, you can use `nodetool.flush()` and `BYPASS CACHE` to ensure that your data comes from sstable, not memtable (I don't know how to make sure that you get something from a memtable or cache, you can be pretty sure, but is there a way to be 100% sure).

Also please add some documentation for this feature, perhaps in docs/cql/cql-extensions.md if you think real users might want to use it, or docs/dev/cql-extensions-internal.md if you think real users shouldn't use it and it's mostly for development and debugging.

Reply:
Yes, I do plan to add much more tests, both positive and negative. I accidentally pushed my "dev" version to this PR (I do push quite often, as a way to save my progress and I usually do that to a separate branch, to not pollute the PR).
I will also add extensive documentation for this. Thanks for reminding me of `cql-extenstions.md`, I forgot about that.

Reply:
We should focus more on the in-tree reference documentation (@annastuchlik). cql-extensions.md is a good place to list the extensions, but the real documentation should be in the reference.

Make sure to slap an "unstable interface, may change at any moment". It's not useful for applications, but people will manage to depend on it.